osbenchmark/workload/params.py

```diff
@@ -1041,11 +1043,28 @@ class VectorSearchPartitionParamSource(VectorDataSetPartitionParamSource):
     PARAMS_NAME_REQUEST_PARAMS = "request-params"
     PARAMS_NAME_SOURCE = "_source"
     PARAMS_NAME_ALLOW_PARTIAL_RESULTS = "allow_partial_search_results"
+    MIN_SCORE_QUERY_TYPE = "min_score"
+    MAX_DISTANCE_QUERY_TYPE = "max_distance"
+    KNN_QUERY_TYPE = "knn"
+    RADIAL_SEARCH_QUERY_RESULT_SIZE = 10000
```

Comment:
nit: DEFAULT_RADIAL_SEARCH_QUERY_RESULT_SIZE

Reply:
is there a way to override this by params?

Reply:
No, as of now I don't see the demand of setting it as parameter